player-computer.js

```diff
@@ -0,0 +1,171 @@
+        answer = await ask("Yes(y) or No(n)")
+    }
+    //successful scenario - answer is yes
+    if (answer.toLowerCase() === "y") {
+        console.log(`Your number was ${randomNumber}! I guessed it!`)
+        process.exit();
+    }
+    //unsuccessful scenario - answer is no
+    while (answer.toLowerCase() === "n") {
+        let highOrLow = await ask("Is it higher (H) or lower (L)?")
+        while (highOrLow !== "h" && highOrLow !== "l") {
+            console.log('Please enter a valid response')
+            highOrLow = await ask("High(h) or Low(l)")
+        }
+
+        if (highOrLow.toLowerCase() === "h") {
+            //making sure user is not cheating
+            if (randomNumber === minNum) {
+                console.log("Your secret number can't be any higher. You are cheating! Good bye!")
+                process.exit()
+            }
+            minNum = randomNumber + 1
+            randomNumber = randomNum(minNum, maxNum)
+            answer = await ask("Is this your number?\n" + randomNumber + " Yes(y) or No(n)")
+            while (answer !== "y" && answer !== "n") {
+                console.log('Please enter a valid response')
+                answer = await ask("Yes(y) or No(n)")
+            }
+        } else if (highOrLow.toLowerCase() === "l") {
+            //making sure user is not cheating
+            if (randomNumber === maxNum) {
+                console.log("Your secret number can't be any lower. You are cheating! Good bye!")
+                process.exit()
+            }
+            maxNum = randomNumber - 1
+            randomNumber = randomNum(minNum, maxNum)
+            answer = await ask("Is this your number?\n" + randomNumber + " Yes(y) or No(n)")
+            while (answer !== "y" && answer !== "n") {
+                console.log('Please enter a valid response')
+                answer = await ask("Yes(y) or No(n)")
+            }
+        }
+        //counting number of tries
+        count += 1;
+    }
+    console.log(`Your number was ${randomNumber}! That was fun! Wasn't it?\nIt took me ${count} tries!\nBye, bye!`)
+    process.exit()
+}
+
+//human guesses
+async function startComputer() {
+    console.log("Let's play a game where I (computer) make up a number and you (human) try to guess it.")
+
+    let maxRange = await ask("First, you have to determine the range! from 1 to ...? (Enter a number greater than 1)")
+
+    // making sure the user enters an actual number.
+    while (isNaN(maxRange)) {
+        maxRange = await ask("That's not a valid response! Give me an actual number! (Enter a number greater than 1)")
+    }
+
+    console.log(`And now... I will pick a number within 1 and ${maxRange}!`)
+    //converting the input-string into a number  
+    let maxR = parseInt(maxRange)
+    maxNum = maxR
+    let computerNum = randomComNum(minNum, maxNum)
+    //console log to make sure the program is working
+    //console.log(computerNum)
```

Comment:
Example of things not being used that could just be removed to cut down on bloat